tools/tests/tester/start.test.ts

```diff
@@ -78,15 +77,15 @@ test('extract_output_values', () => {
     d: 'payload.a.arr[0].d',
     e: 'payload.a.arr[1].e'
   }
-  expect(extract_output_values(response, output1)).toEqual(passed_output({
+  expect(ChapterOutput.extract_output_values(response, output1)).toEqual(passed_output({
```

Comment:
A `ChapterOutput` is something that is the result of an `Output` (a schema) and a response (values). I think the OO way should be closer to this:

```
new ChapterOutput(output, values)
```